src/MSXMotherBoard.cc

```diff
@@ -127,6 +127,7 @@ class StoreSetupCmd final : public Command
 	void tabCompletion(std::vector<string>& tokens) const override;
 private:
 	MSXMotherBoard& motherBoard;
+	const std::map<std::string, MSXMotherBoard::SetupDepth> depthOptionMap;
```

Comment:
No need to change this now. We can tweak this later.

Detail: this could be a global constant (`static const std::map<...>` somewhere in `MSXMotherBoard.cc`) it doesn't need to be a member variable.
Even more a detail: because the full content of the map is known and can't change we could fully initialize it at compile-time. (Now we do run-time allocation of the internal map-nodes and strings).